Running in wrapper mode for '204'

Running python job.

[name: "CPU-QUOTA"
int_val: 15
int_val: 45
int_val: 34
int_val: 76
int_val: 86
int_val: 30
int_val: 32
, name: "DISK"
int_val: 33
int_val: 38
int_val: 16
int_val: 7
int_val: 26
int_val: 5
int_val: 7
, name: "MEMORY"
int_val: 17
int_val: 37
int_val: 9
int_val: 40
int_val: 12
int_val: 7
int_val: 21
, name: "NET"
int_val: 23
int_val: 38
int_val: 22
int_val: 24
int_val: 7
int_val: 9
int_val: 16
]
The paramters are {u'CPU-QUOTA': array([15, 45, 34, 76, 86, 30, 32]), u'NET': array([23, 38, 22, 24,  7,  9, 16]), u'DISK': array([33, 38, 16,  7, 26,  5,  7]), u'MEMORY': array([17, 37,  9, 40, 12,  7, 21])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 112500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 194560.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 31876710.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 1387500000.0
Recovering the memory utilization
Setting cpu quota to 37500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 117760.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 27682406.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 637500000.0
Recovering the memory utilization
Setting cpu quota to 85000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 112640.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 13421772.8
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 337500000.0
Recovering the memory utilization
Setting cpu quota to 37500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 117760.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 27682406.4
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 637500000.0
Recovering the memory utilization
Setting cpu quota to 85000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 112640.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 13421772.8
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 337500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f40384ecb90>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.19 seconds.
